libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -722,7 +724,25 @@ namespace SohImGui {
             }
 
             if (ImGui::BeginMenu("Controller"))
-            {
+	    {
+		ImGuiIO& io = ImGui::GetIO();
+		ImGui::CheckboxFlags("Use Controller Navigation",     &io.ConfigFlags, ImGuiConfigFlags_NavEnableGamepad);      
+
+		ImGui::Separator();
+
+                EnhancementCheckbox("Dpad Support on Pause and File Select", "gDpadPauseName");
+                EnhancementCheckbox("DPad Support in Ocarina and Text Choice", "gDpadOcarinaText");
+                EnhancementCheckbox("DPad Support for Browsing Shop Items", "gDpadShop");
```

Comment:
Looks like you may have forked off a while ago, this reverts back to the old "Dpad/DPad" spelling which was fixed previously in #306.

Reply:
I had pulled an old merge, then corrected, but must have copied the old code. will fix.